.github/workflows/mathlib4docs.yml

```diff
@@ -43,6 +43,6 @@ jobs:
       - name: clean up
         if: always()
         run: |
-          rm -rf workaround mathlib4_docs deploy_key
+          rm -rf ../workaround ../mathlib4_docs
```

Comment:
We need to do this cleanup at the beginning as well.  Otherwise it fail after merging as well.